www/Themes/Centreon-2/style.css

```
@@ -569,6 +569,7 @@ p.description a, p.description a:visited {
 .pending    {background-color: #2AD1D4;}
 .service_unknown    {background-color: #bcbdc0;}
 .info {background-color: #00bfb3;}
+.ack {background-color: #fcf17f;}
```

I think it's more appropriate to use yellow color of acknowledge icon (instead of row)
This will be more readable with a white font
```suggestion
.ack {background-color: rgb(170, 156, 36);}
```

---

not sure, that it's consistent as all color code are hexa... I suggest using uppercase on all hexa code instead

---

Yeah I've hesitating regarding colors, but in fact you're right it will be easier to read :+1: 

I've just pushed the commit thanks